src/main/xml/steps/steps.xml

```diff
@@ -93,6 +93,18 @@ these steps may be found in
   </listitem>
 </itemizedlist>
 
+<para>Options values are often expressed using <link
```

Comment:
Shouldn’t it read “Option values“ or “Options’ values“ instead of “Options values“?